content/docs/guides/hedgedoc.md

```diff
@@ -0,0 +1,144 @@
+# HedgeDoc
+
+ Use Pomerium to secure access to HedgeDoc by providing identity and context. 
+
+ ## What is HedgeDoc?
+
+[HedgeDoc](https://hedgedoc.org/) is a collaborative, web-based Markdown editor that allows you to create notes, graphs, and diagrams in your browser. 
+
+You can share a link to your document with other members of your organization so you can collaborate in real time.
+
+ ## Integrate Pomerium with HedgeDoc
+ 
+ HedgeDoc is open-source, self-hosted software that allows you to configure authentication and authorization with a [reverse proxy](https://docs.hedgedoc.org/guides/reverse-proxy/). 
+ 
+ You can configure Pomerium so that it authenticates and authorizes users against an Identity Provider (IdP) before users can access HedgeDoc's services.
```

Comment:
```suggestion
You can configure Pomerium so that it authenticates and authorizes users against an Identity Provider (IdP) before users can access HedgeDoc's services.
```